test/runtests.jl

```diff
@@ -140,3 +140,21 @@ end
     @test (identity.(Not(1)) .=> [sin, cos] .=> Not(1)) ==
           [InvertedIndices.BroadcastedInvertedIndex(Not(1)) => sin => InvertedIndices.BroadcastedInvertedIndex(Not(1)), InvertedIndices.BroadcastedInvertedIndex(Not(1)) => cos => InvertedIndices.BroadcastedInvertedIndex(Not(1))]
 end
+
+@testset "tuples" begin
+    if VERSION >= v"1.7-dev"
```

Comment:
what happens on earlier Julia versions?

Reply:
Regular indexing of namedtuples is only supported in 1.7+: `nt[(:a, :b)]`. So, indexing with `Not(...)` also doesn't work on earlier versions.

Reply:
Do you know why the decision was that the following does not work:
```
julia> (a=1,b=2,c=3)[(1, 2)]
ERROR: MethodError: no method matching getindex(::NamedTuple{(:a, :b, :c), Tuple{Int64, Int64, Int64}}, ::Tuple{Int64, Int64})
```

Reply:
No idea... But it doesn't work for any collection, not just NamedTuples:
```
julia> (1, 2, 3)[(1, 2)]
ERROR: MethodError: no method matching getindex(::Tuple{Int64, Int64, Int64}, ::Tuple{Int64, Int64})

julia> [1, 2, 3][(1, 2)]
ERROR: ArgumentError: invalid index: (1, 2) of type Tuple{Int64, Int64}
```

Reply:
Yes, but why does then `(:a, :b)` work for `NamedTuple`s?

Reply:
Most likely because it is non-allocating, but then `(1,2)` would be also natural to work.

Reply:
Note that `1.7-dev` is not how development versions of Julia are numbered; the scheme is `-DEV`, and `VersionNumber`s are case-sensitive. In this case, the precise version to check would be `v"1.7.0-DEV.294"`.

Reply:
I think just requiring 1.7 is OK since it has been released.

Reply:
changed to 1.7